src/hadoop-data-node/deploy/hadoop-data-node.yaml.template

```diff
@@ -52,6 +52,8 @@ spec:
           periodSeconds: 3
         resources:
           limits:
+            memory: "4Gi"
+          requests:
             memory: "1Gi"
         env:
         - name: HADOOP_DATANODE_OPTS
```

Comment:
Need to raise JVM memory opts?

Reply:
Not sure, currently we haven't found any java limit was hit.

Reply:
#1658 For 175node screenshot, jave OOM is recorded. So I think a larger JVM opts is needed.

Reply:
Hi @mzmssg , 
What about increase the max heap size to 2G?